spec/features/group_show_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe 'Group show', type: :feature do
+  before(:each) do
+    visit user_session_path
+
+    valid_image = Rack::Test::UploadedFile.new("#{Rails.root}/app/assets/images/test-logo.png")
+
+    @user = User.create(name: 'Tested User', email: 'test@mail.com', password: 'foobar123##')
+    @group1 = Group.create(name: 'Test Group', image: valid_image, user_id: @user.id)
+
+    @bill1 = Bill.create(name: 'Test Bill1', amount: 100, category: @group1, due_date: Date.today, user_id: @user.id)
+    @bill2 = Bill.create(name: 'Test Bill2', amount: 200, category: @group1, due_date: Date.today, user_id: @user.id)
+
+    fill_in 'Email', with: @user.email
+    fill_in 'Password', with: @user.password
+    click_button 'Log in'
+
+    visit group_path(@group1)
+  end
```

Comment:
* Kindly fix your tests as 2 are failing, below ⬇️ is a screenshot 
![image](https://user-images.githubusercontent.com/52497246/173336974-cf8f34db-93e5-4780-ba02-d1ab5086a456.png)